src/main/helm/artemis/templates/_helpers.tpl

```diff
@@ -99,6 +99,39 @@ initContainers:
   volumeMounts:
   - name: etc-override
     mountPath: /tmp/etc-override
+- name: set-users
+  image: {{ .Values.initContainerImage.repository }}:{{ .Values.initContainerImage.tag }}
+  imagePullPolicy: {{ .Values.initContainerImage.pullPolicy}}
+  env:
+    {{- $releaseName := .Release.Name }}
+    {{- range $user, $properties := .Values.users }}
+    - name: ARTEMIS_USER_PW_{{ $user | upper | replace "-" "_" }}
+      valueFrom:
+        secretKeyRef:
+          name: {{ $releaseName }}-{{ $properties.secretName  }}
```

Comment:
the release prefix should be optional, please add additional bool variable like prefixRelease to let us choose which option we want to use

Reply:
As discussed via chat this isn't really needed here and has been removed.